disco/extensions/upgrade_simulation/upgrades/common_functions.py

```diff
@@ -220,6 +222,7 @@ def get_scenario_name(enable_pydss_solve, pydss_volt_var_model):
     return scenario
 
 
+@track_timing(timer_stats_collector)
 def change_pv_pctpmpp(dc_ac_ratio):

```

Comment:
This is the main function where the PctPmpp is changed. It is called in the reload function